news/2020-11-15-osu-mapping-olympiad-7-concludes.md

```diff
@@ -0,0 +1,55 @@
+---
+layout: post
+title: "osu! Mapping Olympiad #7: Results (osu!mania)"
+date: 2020-11-15 18:00:00 +0000
+---
+
+Here come the results of the 2nd edition of the osu!mania Mapping Olympiads, including sick surprises for sure!
+
+![](/wiki/shared/news/banners/osu_mapping_olympiad.jpg)
+
+For the first time in the history of osu!mania Mapping Olympiad, two Olympians have received the same score, hence shared the first place! Please give a huge round of applause to **[Kawawa](https://osu.ppy.sh/users/4647754)** and **[Toaph Daddy](https://osu.ppy.sh/users/7616811)**!
+
+During this iteration, contestants had to create a full mapset on one of 5 songs from several artists, featuring [PUP](https://osu.ppy.sh/beatmaps/artists/79), [HyuN](https://osu.ppy.sh/beatmaps/artists/33), [IAHN](https://osu.ppy.sh/beatmaps/artists/3), [Frums](https://osu.ppy.sh/beatmaps/artists/68) and [Cranky](https://osu.ppy.sh/beatmaps/artists/23)! You can find more details about this contest's criteria over [here](https://osu.ppy.sh/community/contests/97)!
+
+The leaderboards of the 10 best entries are shown right below!
+
+| Rank | Username | Entry Name | Rankability (45) | Song Interpretation (60) | Originality (45) | Hitsounding (15) | Judge Impression (30) | Total Score |
```

Comment:
the "entry name" column is useless. better to delete it or replace it with the song name/map link

Reply:
i prefer to remove it, it would look weird having the song name imo